cmsl1t/collections/efficiency.py

```diff
@@ -1,8 +1,8 @@
 """
-.. module:: collections.efficiency
-    :synopsis: Module for creating efficiency(turnon)-curves
+    .. module:: collections.efficiency
+        :synopsis: Module for creating efficiency(turnon)-curves
 
-.. moduleauthor:: Luke Kreczko
+    .. moduleauthor:: Luke Kreczko
```

Comment:
we should not be keeping those. The [DOI](https://zenodo.org/record/438033#.WSRMhxOGPaY) will include all collaborators.
Only `cmsl1t//collections/efficiency.py` contains it at the moment, so no reason to updated the PR as the file will be removed in the near future.